installer/installer.py

```diff
@@ -576,9 +671,51 @@ def setup_cosmovisor(self):
         
             self.log(f"Changing directory ownership for Cosmovisor to {DEFAULT_CHEQD_USER} user")
             self.exec(f"chown -R {DEFAULT_CHEQD_USER}:{DEFAULT_CHEQD_USER} {self.cosmovisor_root_dir}")
+            self.reload_cosmovisor_systemd()
         except:
             failure_exit(f"Failed to setup Cosmovisor")
 
+    def set_env_vars(self, env_var_name, env_var_value):
+        if not self.check_if_env_var_already_set(env_var_name) and not self.is_default_shell_fish():
+            self.write_to_bashrc(env_var_name, env_var_value) # write to current user
+            self.write_to_bashrc(env_var_name, env_var_value, True) # write to cheqd user .bashrc
+        elif not self.check_if_env_var_already_set(env_var_name) and self.is_default_shell_fish():
+            self.exec(f'set -Uxg {env_var_name} {env_var_value}')
+        else:
+            self.log(f"ENV var {env_var_name} already set")
+
+    def write_to_bashrc(self, env_var_name, env_var_value, is_user_cheqd=False):
```

Comment:
`is_user_cheqd` will always be False, but I assume this is work-in-progress. Ideally, we want to set it for the `cheqd` user AND whichever user is executing the script. This will ensure a start from _either_ user refers to the same config (for example, `cheqd-noded start` right now results in very different things if done as root vs done as cheqd).